tests/unit/Term/AliasGroupListTest.php

```diff
@@ -191,12 +184,10 @@ public function testGivenEmptyGroup_setGroupRemovesGroup() {
 			new AliasGroup( 'en', array( 'foo' ) ),
 		) );
 
-		$expectedList = new AliasGroupList();
-
 		$list->setGroup( new AliasGroup( 'en' ) );
 		$list->setGroup( new AliasGroup( 'de' ) );
 
-		$this->assertEquals( $expectedList, $list );
+		$this->assertTrue( $list->isEmpty() );
```

Comment:
There is only 1 assertion in this test. All you need to know is the name of the test.

We shared this argument already, at least one time. I think relying on the dark magic of `assertEquals` is bad and should be avoided whenever possible. Very similar to us avoiding the dark magic of PHP's `==`.

Adding an `equals` method does not change the argument that all this test needs to know is if the element was removed, which is perfectly expressed by a check for `isEmpty`.